README.md

```diff
@@ -0,0 +1,2 @@
+# exchangebypasslollipop
+exchange bypass for lollipop
```

Comment:
: [PATCH] Create README.md

---
 README.md | 2 ++
 1 file changed, 2 insertions(+)
 create mode 100644 README.md

diff --git a/README.md b/README.md
new file mode 100644
index 0000000..a3bf581
--- /dev/null
+++ b/README.md
@@ -0,0 +1,2 @@
+# exchangebypasslollipop
+exchange bypass for lollipop